.buildkite/pipeline-resource-definitions/locations.yml

```diff
@@ -45,3 +45,4 @@ spec:
     - https://github.com/elastic/kibana/blob/main/.buildkite/pipeline-resource-definitions/trigger-version-dependent-jobs.yml
     - https://github.com/elastic/kibana/blob/main/.buildkite/pipeline-resource-definitions/kibana-pointer-compression.yml
     - https://github.com/elastic/kibana/blob/main/.buildkite/pipeline-resource-definitions/kibana-codeql.yml
+    - https://github.com/elastic/kibana/blob/main/.buildkite/pipeline-resource-definitions/kibana-console-definitions-sync.yml
```

Comment:
This won't have any effect on 8.x - there can only be one definition per pipeline.  

If you want to run on multiple branches the configuration at https://github.com/elastic/kibana/blob/f7a274a4d6e14d4ef6708071b93086e7604934dc/.buildkite/pipeline-resource-definitions/kibana-console-definitions-sync.yml#L25 can be updated, and a cron can also be added in the scheduled section.

Reply:
got it! will close it down then